cmd/help.go

```diff
@@ -1,24 +0,0 @@
-package cmd
-
-import "context"
-
-type Help struct {
-}
-
-func (h Help) Run() error {
```

Comment:
I removed the dedicated help command because Cobra will print usage information multiple ways OOTB:

```
$ attestation help
$ attestation vsa --help
$ attestation help vsa
```